src/sentry/models/release.py

```diff
@@ -529,11 +529,14 @@ def __hash__(self):
 
     @staticmethod
     def is_valid_version(value):
+        if any(c in value for c in BAD_RELEASE_CHARS):
```

Comment:
@barkbarkimashark I've seen an error that started when this code landed.
I believe `value` is being passed as `None`.

https://sentry.io/organizations/sentry/issues/3475102471/events/80ca69f107224742b3cb491407e3aa99/?project=1&referrer=slack
<img width="374" alt="image" src="https://user-images.githubusercontent.com/44410/192615679-c4a02e32-a111-4120-85d9-78bf0618021a.png">

I think the Heroku plugin may be sending these:
<img width="366" alt="image" src="https://user-images.githubusercontent.com/44410/192615960-9d79f01c-4e75-458c-a57c-a709932d50d2.png">